medical/models.py

```diff
@@ -164,12 +166,14 @@ class Service(VersionedModel, ItemOrService):
     code = models.CharField(db_column='ServCode', max_length=6)
     name = models.CharField(db_column='ServName', max_length=100)
     type = models.CharField(db_column='ServType', max_length=1)
+    packagetype = models.CharField(db_column='ServPackageType', max_length=1, default="S")
```

Comment:
FYI, a textchoice class will be more restrictive therefore avoid errors

here an example https://github.com/openimis/openimis-be-payroll_py/blob/3a862282e081356c1fa03a6868c606c294855894/payroll/models.py#L10